tests/src/python/test_qgsrange.py

```diff
@@ -212,14 +214,16 @@ def testOverlaps(self):
         self.assertFalse(range.overlaps(QgsIntRange(-1, 0)))
         self.assertFalse(range.overlaps(QgsIntRange(-10, -1)))
         self.assertFalse(range.overlaps(QgsIntRange(11, 12)))
+        self.assertFalse(range.overlaps(QgsIntRange(10, 12, False, True)))
+        self.assertFalse(QgsIntRange(10, 12, False, True).overlaps(range))
 
         # includes neither end
         range = QgsIntRange(0, 10, False, False)
         self.assertTrue(range.overlaps(QgsIntRange(1, 9)))
         self.assertTrue(range.overlaps(QgsIntRange(1, 10)))
         self.assertTrue(range.overlaps(QgsIntRange(1, 11)))
         self.assertTrue(range.overlaps(QgsIntRange(0, 9)))
-        self.assertTrue(range.overlaps(QgsIntRange(0, 10)))
+        self.assertFalse(range.overlaps(QgsIntRange(0, 10)))
```

Comment:
This is wrong -- these two ranges DO overlap

Reply:
That's right, I have to fix this